substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/Target_jdk_jfr_internal_EventWriter.java

```diff
@@ -29,9 +29,7 @@
 
 @TargetClass(value = jdk.jfr.internal.EventWriter.class, onlyWith = JfrEnabled.class)
 public final class Target_jdk_jfr_internal_EventWriter {
-    @Alias
-    @SuppressWarnings("unused")
-    boolean notified;
+    @Alias @SuppressWarnings("unused") boolean notified;
```

Comment:
Is this formatting change intentional? Looks different from line below but not sure if it's based on line length.

Reply:
Yes, the formatting difference is on purpose (field vs. method/constructor)